genrl/deep/agents/ddpg/ddpg.py

```diff
@@ -293,31 +289,31 @@ def update_params(self, update_interval: int) -> None:
             state, action, reward, next_state, done = (x.to(self.device) for x in batch)
 
             # freeze critic params for policy update
-            for param in self.ac.critic.parameters():
-                param.requires_grad = False
-
-            self.optimizer_policy.zero_grad()
-            self.optimizer_q.zero_grad()
+            # for param in self.ac.critic.parameters():
+            #     param.requires_grad = False
 
-            loss = self.get_q_loss(state, action, reward, next_state, done)
-            loss.backward()
+            # self.optimizer_policy.zero_grad()
+            # self.optimizer_q.zero_grad()
 
-            self.optimizer_q.step()
-            self.optimizer_policy.step()
+            # loss = self.get_q_loss(state, action, reward, next_state, done)
+            # loss.backward()
 
-            # self.optimizer_q.zero_grad()
-            # loss_q = self.get_q_loss(state, action, reward, next_state, done)
-            # loss_q.backward()
             # self.optimizer_q.step()
+            # self.optimizer_policy.step()
```

Comment:
There are too many comments. Makes it more confusing as they're actually used later on.

Reply:
I'll remove the comments.